omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -156,25 +274,15 @@ def __init__(self, task, scene, robot, teleport=False):
         self.robot_model = self.robot.model_name
         self.robot_base_mass = self.robot._links["base_link"].mass
         self.teleport = teleport
-        # self._set_joint_velocities()
-
-    # Set joint velocities for the robots so they move at appropriate speeds.
-    # Not speeding up joints because joint controller is position controller. Asking Josiah about this 
-    def _set_joint_velocities(self):
-        control_idx = np.concatenate([self.robot.trunk_control_idx, self.robot.arm_control_idx[self.arm]])
         if self.robot_model == "Tiago":
-            joint_gains = [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
-            joints = np.array([joint for joint in self.robot.joints.values()])
-            arm_joints = joints[control_idx]
-            for i, joint in enumerate(arm_joints):
-                pass
-                # damps = []
-                # joint.set_control_type(ControlType.POSITION, kp=10000000.0)
-                # print(joint.name)
-                # for dof_handle, dof_property in zip(joint._dof_handles, joint._dof_properties):
-                #     damps.append(dof_property.stiffness)
-                # print(damps)
-                # print("-------")
+            self._setup_tiago()
+
+    # Disable grasping frame for Tiago robot (Should be cleaned up in the future)
+    def _setup_tiago(self):
+        for link in self.robot.links.values():
+            for mesh in link.collision_meshes.values():
+                if "grasping_frame" in link.prim_path:
+                    mesh.collision_enabled = False
```

Comment:
If we are doing this here, let's add the same thing for Fetch to update the base mass. Also let's open tickets on this repo for fixes to this if there's something for the OG team to do.

Reply:
@sujaygarlanka

Reply:
Re ran tiago in the sim, and grasping frame is there but collisions are not enabled. Need to look into why it showed up earlier when I was testing. Going to leave this code it for now to make sure it is not an issue. Will investigate and if I can't replicate it again, will remove it. If I can replicate, I'll file an issue with the OG team.

Reply:
Fetch's base mass should only be higher when it moves its hand. Higher base mass interferes with its navigation, so it should probably not be in the setup, if that is what you mean by update the base mass.